service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -187,12 +184,9 @@ public static LagomClientFactory create(String serviceName, ClassLoader classLoa
         EventLoopGroup eventLoop = new NioEventLoopGroup();
 
         // WS
-        WSClientConfig wsClientConfig = new WSConfigParser(configuration, environment).parse();
-        AhcWSClientConfig ahcWSClientConfig = new AhcWSClientConfigParser(wsClientConfig, configuration, environment).parse();
-        new SystemConfiguration().configure(wsClientConfig);
-        DefaultAsyncHttpClientConfig.Builder configBuilder = new AhcConfigBuilder(ahcWSClientConfig).configure();
-        configBuilder.setEventLoopGroup(eventLoop);
-        WSClient wsClient = new AhcWSClient(configBuilder.build(), materializer);
+        WSClientConfig wsClientConfig = new WSConfigParser(configuration.underlying(), environment.classLoader()).parse();
+        AhcWSClientConfig ahcWSClientConfig = new AhcWSClientConfigParser(wsClientConfig, configuration.underlying(), environment.classLoader()).parse();
+        WSClient wsClient = AhcWSClient.apply(ahcWSClientConfig, scala.Option.empty(), materializer);
```

Comment:
See discussion in PR #688
https://github.com/lagom/lagom/pull/688/files/86244a9188dad011402ec4eb63a2f34167e80322#r115656805